rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] A rental belongs to a book and also belongs to a person. To make this relationship possible we need to add an attr_accessor for @book and @person in the Rental class.